processing/src/main/java/io/druid/query/lookup/LookupConfig.java

```diff
@@ -29,22 +29,44 @@
   @JsonProperty
   private final String snapshotWorkingDir;
 
+  @JsonProperty
+  private int numLookupLoadingThreads = 10;
```

Comment:
can you update the documentation with this and also mention that this threadpool is only created at the time of start and destroyed , it is not kept during the lifetime of jvm.